2015-04-25-designs-and-hacks.md

```diff
@@ -0,0 +1,32 @@
+# Designs and Hacks, Apr 25, 2015
+
+Lots of things that we've talking about for the last couple months. Time to push some pixels and code and breath life into these objective. Let’s meet and work on driving the Open Soure Design movement forward. Everyone who wants to contribute is welcome :)
+
+
+## Date & Location
+
+- **Date:** Sun, 25 April 2015
+- **Time:** 14:00 – 18:00 CET (UTC +1:00)
+- **Where:** @bnvk Haus
+	- Gneisenaustraße 52
+	- Kreuzberg, Berlin
+	- Near U7-Südstern stop
+- **Where (remote):** Participate via [video chat](https://appear.in/osd-designs-and-hacks)
```

Comment:
I would say we just use only https://appear.in/opensourcedesign for any and all of our video calls. Then we just have to open it from browser history and it makes it much easier to invite people.